kylin-spark-project/kylin-spark-engine/src/main/java/org/apache/kylin/engine/spark/job/NSparkExecutable.java

```diff
@@ -146,7 +146,7 @@ protected ExecuteResult doWork(ExecutableContext context) throws ExecuteExceptio
             throw new ExecuteException("meta dump failed", e);
         }
         String filePath = dumpArgs();
-        if (config.isUTEnv() || config.isZKLocal()) {
+        if (config.isUTEnv() || config.isLOCALEnv() || config.isZKLocal()) {
```

Comment:
UT is unittest, but local is local. So isLocalEnv()?

Reply:
Agreed. Please capitalize **local** .